runtime/runtime_test.go

```diff
@@ -82,6 +85,14 @@ func (i *testRuntimeInterface) ResolveImport(location Location) ([]byte, error)
 	return i.resolveImport(location)
 }
 
+func (i *testRuntimeInterface) GetCachedProgram(location Location) (*ast.Program, error) {
+	return i.getCachedProgram(location)
+}
+
+func (i *testRuntimeInterface) CacheProgram(location Location, program *ast.Program) error {
+	return i.cacheProgram(location, program)
+}
```

Comment:
Maybe instead of adding empty implementations for `getCachedProgram` and `cacheProgram` to all the tests (e.g. https://github.com/onflow/cadence/pull/37/files#diff-991fa0d56f6a51df18449cd1f05bbbcaR249-R254), maybe just make the two functions optional (and then remove all additions starting with https://github.com/onflow/cadence/pull/37/files#diff-991fa0d56f6a51df18449cd1f05bbbcaR249)

```suggestion
func (i *testRuntimeInterface) GetCachedProgram(location Location) (*ast.Program, error) {
	if i.getCachedProgram == nil {
		return nil, nil
	}
	return i.getCachedProgram(location)
}

func (i *testRuntimeInterface) CacheProgram(location Location, program *ast.Program) error {
	if i.cacheProgram == nil {
		return nil
	}
	return i.cacheProgram(location, program)
}
```

Reply:
Sounds good!